neuralmonkey/decoders/autoregressive.py

```diff
@@ -120,6 +144,8 @@ def batch_size(self) -> tf.Tensor:
     @tensor
     def decoding_w(self) -> tf.Variable:
         with tf.name_scope("output_projection"):
+            if self.tie_embeddings:
+                return tf.transpose(self.embedding_matrix)
```

Comment:
Tady je potřeba kontrolovat, jestli dimenze hidden statů odpovídá dimenzi embeddingů, jinak se to nemůže stát.